docs/madr/decisions/039-transition-to-new-policies.md

```diff
@@ -451,23 +644,3 @@ spec:
   op: add
   path: /type.googleapis.com~1envoy.config.cluster.v3.Cluster/test-cluster
 ```
-
-## HealthCheck -> MeshHealthCheck
-
-TBD
-
-## RateLimit -> MeshRateLimit
-
-TBD
-
-## Retry -> MeshRetry
-
-TBD
-
-## TrafficLog -> MeshTrafficLog
-
-TBD
-
-## TrafficTrace -> MeshTrafficTrace
-
-TBD
```

Comment:
Are these not interesting? Why are we removing them?

Reply:
TBH it takes time to produce meaningful examples so I thought the existing examples were sufficient enough. But maybe I can do a few more, WDYT?

Reply:
we can always add it afterwards, it's fine now